### ISSUE TITLE
so, is this project dead?

### ISSUE THREAD

**Issue by zombiehoffa** [BeardedTek-com/fEVR#113]
No commits since last september leaves me wondering..

**Comment by BeardedTek** [comment]
No, not dead.  I am currently working on a rewrite to take advantage of new features of frigate.

**Comment by bazylhorsey** [comment]
At this point I'm pretty certain Frigate has provided all this functionality.

**Comment by BeardedTek** [comment]
Yes, and I was actually looking to archive this repo.  frigate 0.13 is amazing including the interface.

Thank you.
